### ISSUE TITLE
Set initial value for ng-select

### ISSUE THREAD

**Issue by SachinSharmaEarthport** [basvandenberg/angular2-select#82]
Hello,

Is it possible to set an initial value on the ng-select when the page loads from a value in the model?  For example if my ng-select has the options "1","2" and "3" and I have a model property on my component called "selectedNumber" how can "2" be set as the initial value in ng-init of my component?  

**Page:**
                <ng-select
                        [options]="numbers"
                        [(ngModel)]="selectedNumber"
                        name="number">
                </ng-select>

**Component:**

    numbers:Array<any> = [];
    selectedNumber:any;

     this.numbers.push({value: 1, label: '1'});
     this.numbers.push({value: 2, label: '2'});
     this.numbers.push({value: 3, label: '3'});

Is there any reference/demo I can view for this?

Kind Regards,

Sachin Sharma

**Comment by basvandenberg** [comment]
There are some examples [here](https://basvandenberg.github.io/angular2-select/).

The first example in [this](https://basvandenberg.github.io/angular2-select/#value) section shows that you can change the value of your `selectedNumber` variable to update the current selection. So in your `ngOnInit` you could add:

```
ngOnInit() {
    this.selectedNumber = '2';
}
```

**Comment by SachinSharmaEarthport** [comment]
This worked when my value is set as a string like on your example above.  I tried setting it as a number i..e-

```
ngOnInit() {
    this.selectedNumber = 2;
}
```

But could not get this working.  Is it possible to do?

**Comment by Andikki** [comment]
The interface for Option is `{value: string, label: string}`. Based on that, you shouldn't be working with numbers to begin with.

**Comment by shujahm** [comment]
I have submitted a pull request where you can attribute your initial value so that when the control loads up respective value is selected. Waiting for the pull request to be accepted: 
https://github.com/basvandenberg/angular2-select/pull/84

**Comment by basvandenberg** [comment]
Closed #84. Set values using strings.
